packages/@azure/keyvault/typedoc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "mode": "file",
+  "out": "typedoc",
+  "readme": "none",
+  "excludeNotExported": true,
+  "excludePrivate": true,
+  "exclude": [
+    "lib/generated/**/*",
```

Comment:
SecretAttributes is defined in lib/generated, but is re-exported through secretModels.ts.  However, the type is not in the generated reference docs.  Possibly this is because of this generic exclude.  Is there a way we can explicitly include SecretAttributes but still exclude the remaining generated code?

Reply:
This may be hard. I haven't seen any typedoc doc regarding excluding types.  Let me explore a bit more.

We are having issue with `ServiceClientCredentials` and `Pipeline` used in `SecretsClient` constructor too. Could be a different issue though.

Reply:
There isn't good support in `typedoc` for export declarations yet. I could manually generate the docs using the workaround mentioned in https://github.com/TypeStrong/typedoc/issues/639#issuecomment-465874272. `Pipeline` would be included, but `ServiceClientCredentials` would not because it's coming from `ms-rest-js`.